### PR TITLE
FIX PINK-863: split ec2 and savingplans permissions

### PR DESCRIPTION
**Pull request by mencnert**

### CODE REVIEW

File: modules/vertice-governance-role/variables.tf [hunk 1]
```diff
@@ -58,6 +58,8 @@ variable "billing_policy_addons" {
     es_ri          = optional(bool, true),
     rds_ri         = optional(bool, true),
     redshift_ri    = optional(bool, true),
+    # why also here?
+    savingplans    = optional(bool, true),
```
Comment (kristian-lesko) [inline modules/vertice-governance-role/variables.tf:62]: Can we follow AWS's naming of the feature in the variable name, please?
```suggestion
    savings_plans    = optional(bool, true),
```

File: variables.tf [hunk 2]
```diff
@@ -60,6 +60,7 @@ variable "billing_policy_addons" {
     es_ri          = optional(bool, true),
     rds_ri         = optional(bool, true),
     redshift_ri    = optional(bool, true),
+    savingplans    = optional(bool, true),
```
Comment (kristian-lesko) [inline variables.tf:63]: also here

File: modules/vertice-governance-role/variables.tf [hunk 3]
```diff
@@ -58,6 +58,8 @@ variable "billing_policy_addons" {
     es_ri          = optional(bool, true),
     rds_ri         = optional(bool, true),
     redshift_ri    = optional(bool, true),
+    # why also here?
```
Comment (kristian-lesko) [inline modules/vertice-governance-role/variables.tf:61]: The variable needs to propagate to the `modules/vertice-governance-role` submodule, which is called from the root module, hence both places need the variable.

File: modules/vertice-governance-role/iam_policies.tf [hunk 4]
```diff
@@ -15,6 +15,7 @@ locals {
     "cur:Describe*",
     "organizations:Describe*",
     "organizations:List*",
+    # should those two be here?
```
Comment (kristian-lesko) [inline modules/vertice-governance-role/iam_policies.tf:18]: We could delete this in a major release if not needed.

**Review by kristian-lesko — DISMISSED** [review]

**Review by PerekhrestB — APPROVED** [review]

**Pull request merged**